Selenium.Axe/Selenium.Axe.Test/AxeBuilderTest.cs

```diff
@@ -100,7 +104,8 @@ public void ShouldPassContextIfIncludeSet()
             jsExecutorMock.VerifyAll();
         }
 
-        [TestMethod]
+        [Test]
+        [NonParallelizable]
```

Comment:
Is marking this `[NonParallelizable]` a side effect of having used shared mock instances, or is there another reeason for it that I'm not seeing?